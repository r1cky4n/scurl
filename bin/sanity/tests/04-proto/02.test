--tlsv1.1 https://sha256.badssl.com/
