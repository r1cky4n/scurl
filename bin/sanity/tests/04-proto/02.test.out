<!DOCTYPE html>
<html>
<head>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="shortcut icon" href="/icons/favicon-green.ico"/>
  <link rel="apple-touch-icon" href="/icons/icon-green.png"/>
  <title>sha256.badssl.com</title>
  <link rel="stylesheet" href="/style.css">
  <style>body { background: green; }</style>
</head>
<body>
<div id="content">
  <h1 style="font-size: 12vw;">
    sha256.<br>badssl.com
  </h1>
</div>

</body>
</html>
